.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,61 @@
+name: Build container image, publish as Github-package
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+on:
+  push:
+    branches: [ master, develop ]
```

Comment:
Our main release branch is `main`, instead of `master` -- think we'll want to change this?

Reply:
Right, I knew that.